### ISSUE TITLE
[w113] Access Undefined Global Variable "module"

### ISSUE THREAD

**Issue by cyw3** [mpeterv/luacheck#174]
The code on the top of lua file:

`module("mybit", package.seeall)`

mybit.lua:1:1: accessing undefined variable 'module'

I think it is a false alarm.

**Comment by mpeterv** [comment]
On master branch by default globals of all Lua versions are allowed. This should get rid of these "false positives" when checking random files outside projects, for projects it's a good idea to have a config (`.luacheckrc`) and specify which version of Lua the project uses. This way luacheck warns about using globals that are only available for other versions.

For example:

```lua
-- .luacheckrc in the root directory of the project
std = "luajit"
```

See `luacheck --help` or documentation for the list of available sets of standard globals (stds) for different Lua versions.
